<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteList</class>
 <widget class="QWidget" name="NoteList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item>
      <widget class="QToolButton" name="actionsButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="noteContainer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>212</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addNoteButton">
     <property name="text">
      <string>Add a note...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addNoteButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteList</receiver>
   <slot>addNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionsButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteList</receiver>
   <slot>showActions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNote()</slot>
  <slot>showActions()</slot>
 </slots>
</ui>
